karma.conf.js

```diff
@@ -139,8 +139,7 @@ module.exports = function(config) {
     // test results reporter to use
     // possible values: 'dots', 'progress'
     // available reporters: https://npmjs.org/browse/keyword/karma-reporter
-    reporters: ['spec', 'coverage-istanbul'],
-    specReporter: {showSpecTiming: true, suppressPassed: true, suppressSkipped: true},
+    reporters: ['progress', 'coverage-istanbul'],
```

Comment:
This reporter is more friendly for dev (it only shows one line with `numberDone/totalNumberOfTest`)